analyzer-core/src/test/java/ffb/analyzer/core/pubsub/ExampleSubscriber.java

```diff
@@ -0,0 +1,14 @@
+package ffb.analyzer.core.pubsub;
+
+import org.junit.Assert;
+
+import java.util.Map;
+
+public class ExampleSubscriber implements Subscriber {
```

Comment:
My vote is we should move this as an inner class to `PubSubTests`. That way, it's clear this is a class highly associated with those tests.

Reply:
That makes sense. Updated

Reply:
I agree.